src/domUpdates.js

```diff
@@ -32,6 +38,24 @@ const domUpdates = {
         });
     },
 
+    displayAvailableRooms(rooms) {
+        const allRoomsSection = document.getElementById('rooms');
+        allRoomsSection.innerHTML = '';
+        rooms.forEach(room => {
+            allRoomsSection.innerHTML += `
+            <article id="roomDisplay">
+                <h1 id="roomType">${room.roomType}</h1>
+                <li id="roomNum">${room.number}</li>
+                <li id="bidet">${room.bidet}</li>
```

Comment:
Hell yeah! My only thought here is to consider finding ways of capitalizing things like roomType! (not needed per se, and definitely adjustable to style, but let me know if you need help doing it)

Reply:
That is super weird... my version has a completely different code here. That doesn't make sense why this pushed up, and not the version I have... Github can be maddening sometimes.